README.md

```diff
@@ -92,6 +92,8 @@ identifier" that was printed by `add_key`.  The directory must be empty.
 # (Alternatively, use `tune2fs -O encrypt` on an existing ext4 filesystem.)
 # (For f2fs, use `mkfs.f2fs -O encrypt` or `fsck.f2fs -O encrypt`.)
 > mkfs.ext4 -O encrypt /dev/vdb
+# To make use of inline crypto hardware, use:
+# `mount -o inlinecrypt /dev/vdb /mnt`
```

Comment:
This is a little hard to read since this new comment is inserted into a sequence of shell commands that already has its own comment.  How about breaking up the "create filesystem" and "mount filesystem" steps like this:

```
# Create an ext4 filesystem that supports encryption.                            
# (Alternatively, use `tune2fs -O encrypt` on an existing ext4 filesystem.)      
# (For f2fs, use `mkfs.f2fs -O encrypt` or `fsck.f2fs -O encrypt`.)              
> mkfs.ext4 -O encrypt /dev/vdb                                                  
                                                                                 
# Mount the filesystem.  Optionally add any desired mount options, such as       
# `-o inlinecrypt` to make use of inline crypto hardware.                        
> mount /dev/vdb /mnt                                                            
```